src/org/jitsi/impl/neomedia/RawPacket.java

```diff
@@ -122,6 +122,21 @@ public static boolean isPacketMarked(byte[] buffer, int offset, int length)
     }
 
     /**
+     * Perform checks on the packet represented by this instance and
+     * return <tt>true</tt> if it is found to be invalid. A return value of
+     * <tt>false</tt> does not necessarily mean that the packet is valid.
+     *
+     * @return <tt>true</tt> if the RTP/RTCP packet represented by this
+     * instance is found to be invalid, <tt>false</tt> otherwise.
+     */
+    public static boolean isInvalid(byte[] buffer, int offset, int length)
+    {
+        return (buffer == null)
+            || (buffer.length < offset + length)
+            || (length < FIXED_HEADER_SIZE);
```

Comment:
maybe for a future commit, but it seems like having RTPPacket and RTCPPacket (which are built via a RawPacket maybe?) would be a good place for RTP/RTCP-specific operations?  it could not make a copy and just refer to/parse the backing RawPacket.